src/pyedb/dotnet/edb_core/components.py

```diff
@@ -1335,18 +1335,21 @@ def add_rlc_boundary(self, component=None, circuit_type=True):
             return True
 
     @pyedb_function_handler()
-    def _create_pin_group_terminal(self, pingroup, isref=False, term_name=None):
+    def _create_pin_group_terminal(self, pingroup, isref=False, term_name=None, term_type="gap"):
```

Comment:
Is gap port supported on a pin grroup?

Reply:
Yes. The port created when running this function PinGroupTerminal.Create(...) is a gap port. I added the other options of changing it to circuit port and auto.

Reply:
I wouldn't set "gap" as default. Most of users are dealing with circuit ports